### PR TITLE
Python 3 support

### PR DESCRIPTION
**Pull request by Lartza**
Adds support for Python 3 while maintaining Python 2 support, partly through the help of futurize and the future library.

Cursory testing was done on 2.7 and 3.6 under Windows and using Ice 3.7. At least older Python 3 versions should work too. Support should preferably only be kept for 2.7 as per Python's official porting suggestions.

### CODE REVIEW

**Comment by Lartza** [comment]
https://github.com/alfg/murmur-rest/pull/22/commits/b9f3d6185db9f7a4e98507bf108a5eeac8d99820 fixes an issue with long values not being handled correctly under either Python versions.

Also don't import str so the isinstance check works(that apparently requires unicode_literals)

**Comment by alfg** [comment]
Hi @Lartza, 

Thanks so much for the PR. 👍 I will review and test later tonight.

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
-Flask==0.10.1
-Flask-Classy==0.6.10
-Flask-HTTPAuth==2.2.1
+Flask==0.12.2
+Flask-Classful==0.14.1
```
Comment (alfg) [inline requirements.txt]: 👍

**Comment by alfg** [comment]
Looks good and works! Thank you so much!

**Comment by Lartza** [comment]
I'm sure there might be some lurking bugs there, but next time I have free time I may write and PR some tests so I can work on murmur-rest more :) Also that's what issue reports are for ;)

**Comment by alfg** [comment]
Tests would be great. I haven't updated or maintained this project in a while, so having any tests would definitely be a huge help! I added #23 as a reminder!

**Comment by Lartza** [comment]
Yeah it just worked(tm) for me too until I stopped being able to compile Ice for Python 2 under Arch ;) So that sparked my interest and well since I started I may as well continue when I have the time :P

**Pull request merged**